gitpod-io/gitpod
# [supervisor] Remove unnecessary features when running inside `run-gp`

## Description
This PR introduces changes to supervisor to make it work nicely when running inside `run-gp`. Some changes will actually skip some logic from supervisor which is not required. 


## Related Issue(s)

Fixes #

## How to test
1. Start a workspace https://ide-superv38f4e2b21a.preview.gitpod-dev.com/#https://github.com/andreafalzetti/gitpod-ports
2. Check gp cli works as expected `gp top`, `gp tasks list`, `gp tasks attach`, `gp ports list`
3. Check if notifications work `/.supervisor/supervisor notify "test"`

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
